webapp/Connector/src/controller/FilterStatus.js

```diff
@@ -30,6 +30,7 @@ Ext.define('Connector.controller.FilterStatus', {
                 navfilter.add({
                     xtype: 'container',
                     itemId: 'filterstatuscontainer',
+                    id: 'filterstatuscontainer-id',
```

Comment:
Just curious, what are we using the ID's on these elements for?

Reply:
To debug the dom, otherwise ext assigns its own id which makes it hard to debug at times